### PR TITLE
Add Windows 11 to section about DX12

### PR DESCRIPTION
**Pull request by HeavyRain266**
**Connections**
None.

**Description**
DX12 is API used by Windows 11.

**Testing**
None.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ non-WGSL shaders if you're running on WebGPU.
 
 WGSL is always supported by default, but GLSL and SPIR-V need features enabled to compile in support.
 
-To enable SPIR-V shaders, enable the `spirv` feature of wgpu.
```
Comment (cwfitzgerald) [inline README.md:92]: What happened here? These need to be preserved as they need to be on separate lines.

File: README.md [hunk 2]
```diff
@@ -66,21 +66,21 @@ We have a [wiki](https://github.com/gfx-rs/wgpu/wiki) that serves as a knowledge
...
-  Metal  |                               |                           | :white_check_mark:  |
-  DX12   | :white_check_mark: (W10 only) |                           |                     |
-  DX11   | :hammer_and_wrench:           |                           |                     |
-  GLES3  |                               | :ok:                      |                     |
-  Angle  | :ok:                          | :ok:                      | :hammer_and_wrench: |
+   API   |    Windows                         |  Linux & Android          |    macOS & iOS      |
+  -----  | ---------------------------------- | ------------------------- | ------------------- |
+  Vulkan | :white_check_mark:                 | :white_check_mark:        |                     |
+  Metal  |                                    |                           | :white_check_mark:  |
+  DX12   | :white_check_mark: (W10 & 11 only) |                           |                     |
```
Comment (cwfitzgerald) [inline README.md:73]: Could we do `(W10+)` to save some characters and be future proof.

**Review by cwfitzgerald — CHANGES_REQUESTED** [review]

**Comment by cwfitzgerald** [comment]
Thanks! Just a couple small comments

File: README.md (hunk 1 above)
Comment (HeavyRain266) [reply README.md:92]: Looks like editorconfig messed it up.

File: README.md (hunk 2 above)
Comment (HeavyRain266) [reply README.md:73]: Yeah, sure

**Comment by HeavyRain266** [comment]
Resolved all requests.

**Review by cwfitzgerald — APPROVED** [review]
Thank you so much!

**Comment by cwfitzgerald** [comment]
No idea what is going on with CI, it's not even giving me an option to trigger the jobs

**Comment by HeavyRain266** [comment]
> No idea what is going on with CI, it's not even giving me an option to trigger the jobs

Github is partially down right now.

**Pull request merged**